Neos.Flow/Tests/Functional/ObjectManagement/Fixtures/ValueObjectClassA.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Neos\Flow\Tests\Functional\ObjectManagement\Fixtures;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+/**
+ * A readonly class in the style of a value object
+ */
+readonly class ValueObjectClassA implements \JsonSerializable
+{
+    public function __construct(
+        readonly public string $value,
```

Comment:
sorry nitpicky: i think we can drop the `readonly` in the property then

Reply:
It's to make sure it's really, really, really readonly! 😇

Reply:
Honestly, yes – but I'd mix it, and leave one in. Just to have yet another edge-case…